src/x509.c

```diff
@@ -11632,24 +11681,40 @@ int wolfSSL_X509_check_ip_asc(WOLFSSL_X509 *x, const char *ipasc,
         ret = WOLFSSL_SUCCESS;
     }
 
+#ifdef WOLFSSL_SMALL_STACK
+    if (ret == WOLFSSL_SUCCESS) {
+        dCert = (DecodedCert *)XMALLOC(sizeof(*dCert), x->heap,
```

Comment:
Allocation of DecodedCert should use `DYNAMIC_TYPE_DCERT`

Reply:
done